app/controllers/appointment_controller.js

```diff
@@ -0,0 +1,611 @@
+                action: 'create_time_slot'
+            },
+            relatedParty: {
+                id: faker.random.uuid(),
+                href: faker.internet.url(),
+                name: "string",
+                role: "string",
+                "@referredType": "string"
+            },
+            relatedEntity: {
+                id: faker.random.uuid(),
+                href: faker.internet.url(),
+                name: "string",
+                role: "string",
+                "@referredType": "string"
+            },
+            place: {
+                id: faker.random.uuid(),
+                href: faker.internet.url(),
+                name: "string",
+                role: "string",
+                "@referredType": "string",
+                "@schemaLocation": "string"
+            },
+            requestedTimeSlot: [
+                {
+                    validFor: {
+                        startDateTime: new Date(),
+                        endDateTime: new Date()    
+                    },
+                    relatedParty: {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url(),
+                        name: "string",
+                        role: "string",
+                        "@referredType": "string",
+                        
+                    },
+                }
+            ]    
+        }
+    },
+    index_time_slot: function(request)
+    {
+        console.log("appointment_controller.index_time_slot")
+        return [
+            {
+                id: faker.random.uuid(),
+                routeDetails: {
+                    controller: 'Appointment',
+                    action: 'index_time_slot'
+                },
+                requestedTimeSlot: [
+                    {
+                        validFor: {
+                            startDateTime: new Date(),
+                            endDateTime: new Date()    
+                        }
+                    }
+                ],
+                relatedParty: {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: "string",
+                    role: "string",
+                },
+                place: {
+                    role: "interventionAddress",
+                    "@referredType": "geographicAddress",
+                    geographicAddress: {
+                        streetNr: faker.random.number(),
+                        streetType: "Avenue",
+                        streetName: "string",
+                        postCode: faker.random.number(),
+                        city: "Paris",
+                        country: "France" 
+                    }
+                },
+                relatedEntity: [
+                    {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url(),
+                        name: "Internet connexion problem to solve",
```

Comment:
```suggestion
                        name: "Internet connection problem to solve",
```
There are others that use relatedEntity. Please could you check them for me.

Reply:
done